src/Analyzer/ConfluenceAnalyzer.php

```diff
@@ -128,6 +183,51 @@ protected function doAnalyze( SplFileInfo $file ): bool {
 		$this->dom->load( $file->getPathname() );
 		$this->helper = new XMLHelper( $this->dom );
 
+		if ( !$this->hasAdvancedConfig ) {
```

Comment:
So we require the advanced config to contain _all_ required information, right? No partial config can be passed, making the tool ask for additional input?

That's fine, but we need to document this and maybe provide a `config.sample.yml`.

Reply:
Yes, that's right